src/connected/connectionsetup.ts

```diff
@@ -28,17 +28,27 @@ const TOKEN_RECEIVED_COMMAND = 'tokenReceived';
 const OPEN_TOKEN_GENERATION_PAGE_COMMAND = 'openTokenGenerationPage';
 const SAVE_CONNECTION_COMMAND = 'saveConnection';
 
+export function assistCreatingConnection(context: vscode.ExtensionContext) {
```

Comment:
This will be called when user has tried to open hotspot in IDE but there is no connection configured.